### ISSUE TITLE
Greek helmet appearance

### ISSUE THREAD

**Issue by poratshay** [obophenotype/human-phenotype-ontology#11592]
**Preferred term label:**
Greek helmet appearance

**Synonyms**
Greek warrior helmet appearance

**Definition (free text, please give PubMed ID)**
broad flat nasal bridge and high forehead PMID 25137600

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Wolf-Hirschhorn syndrome

**Your nano-attribution (ORCID)**
0000-0001-9947-035X

**Comment by pnrobinson** [comment]
@poratshay added as HP:6001478
Def: A prenatal sonographic finding characterized by a high forehead, prominent glabella, and a broad, un-tapered nasal bridge that merges directly into the brow line. This appearance is said to resemble a Greek (Corinthian) warrior helmet because the wide, solid vertical column down the center of the face is said to resemble the fixed, protective nose guard of a Corinthian helmet.
